### PR TITLE
refactor: icon to icons for IconButton and Header component

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
This pr refactors the iconbutton component default db icon to the new ant-d dbicon outlined.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before
PostgreSQL

after
PostgreSQL

Screen Shot 2021-07-13 at 2 05 27 PM

### TESTING INSTRUCTIONS
Test out in databasemodal and dashboard (edit view)

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/IconButton/index.tsx [hunk 1]
```diff
@@ -41,6 +41,15 @@ const StyledImage = styled.div`
   height: ${({ theme }) => theme.gridUnit * 18}px;
   margin: ${({ theme }) => theme.gridUnit * 3}px 0;
 
+  .default-db-icon {
+    font-size: ${({ theme }) => theme.gridUnit * 9}px;
```
Comment (suddjian) [inline superset-frontend/src/components/IconButton/index.tsx:45]: Font sizes shouldn't be set using grid units. There is `theme.typography.sizes` for that.

Comment (suddjian) [reply superset-frontend/src/components/IconButton/index.tsx:45]: Looks like we don't have a typography size large enough for what you're going for here though. Doing math with the typography sizes sounds pretty unsavory. If we don't want to add an `xxxl` size maybe grid units are actually the right fit here.

**Review by suddjian — APPROVED** [review]
LGTM, I'm okay with ignoring the sizing semantics issue.

**Comment by suddjian** [comment]
Looks like the e2e tests might need some tweaks to select the new icons correctly.

File: superset-frontend/src/components/IconButton/index.tsx (hunk 1 above)
Comment (pkdotson) [reply superset-frontend/src/components/IconButton/index.tsx:45]: Oh yah forgot about that. Maybe i'll just put down the actual font-size then.

**Comment by hughhhh** [comment]
@yousoph are okay with the default images for the aria update?

**Comment by yousoph** [comment]
Thumbs up on the db icon 👍

**Pull request merged**